### DISCUSSION TITLE
Smart Meter Iskra MT 175 showing multiple values in GUI, but only sending current power consumption via MQTT?

### DISCUSSION THREAD

**Discussion by ojkastl** [arendst/Tasmota#19861] [Q&A]
I have a Tasmota device to read my house's smart meter, which is a `Iskra MT 175`.

The Tasmota GUI shows multiple values for the device, including the current power comsumption and the total sum of kWh consumed since the smart meter was installed.

But via MQTT I only have the current power consumption available via the `tele/tasmota_XXX/SENSOR` topic. 

```
{"Time":"2023-10-28T14:27:38","MT175":{"P":324.00}}
```

I fail to find if there is another topic that would show me the other values, too?

Configuration is done with the script from the docs:
https://tasmota.github.io/docs/Smart-Meter-Interface/#iskra-mt-175-sml

Thanks in advance!

**Comment by gemu2015** [comment]
The descriptor is configured to send current consumption immediately while total_in is configured to sent on Teleperiod time which can be set via teleperiod cmd to a minimum of 10 seconds
You may modify the decoder lines last parameter which is decimal precision. If you add 16 to decimal precision, value is sent immediately else on teleperiod time

**Reply by ojkastl** [reply]
Thanks for the answer, that would explain why I did not see the total consumption. I'll have a look at the settings and see, whether I want to change them...
